packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-container.jsx

```diff
@@ -14,7 +14,7 @@ import React from 'react';
 import Unsupported from 'Components/poi/status/unsupported';
 import UploadComplete from 'Components/poi/status/upload-complete';
 import Verified from 'Components/poi/status/verified';
-import { populateVerificationStatus } from '../Helpers/verification';
+import { populateVerificationStatus } from '../Helpers/verification.js';
```

Comment:
Why is the file extension added here?

Reply:
I think, I added it accidentally while returning this file from TS back to JS.
thanks